data/brands/office/insurance.json

```
@@ -856,17 +856,7 @@
       "displayName": "Groupama",
       "id": "groupama-924bc9",
       "locationSet": {
-        "include": [
-          "bg",
-          "cn",
-          "gr",
-          "hu",
-          "it",
-          "ro",
-          "sk",
-          "tr"
-        ]
-      },
+        "include": [ "bg", "cn", "fr", "gr", "hu", "it", "ro", "sk", "tr"]},
       "matchTags": ["amenity/bank|groupama"],
```

`matchTags` are used to match existing features with different tags to this entry. If Groupma locations tagged with `amenity=bank` should be changed to `office=insurance`, the suggestion below should be committed to use correct formatting (see [this wiki page](https://github.com/osmlab/name-suggestion-index/wiki/Contributing#matchnamesmatchtags-optional))

```suggestion
       "matchTags": ["amenity/bank"],
```